scss/_book-list.scss

```diff
@@ -0,0 +1,39 @@
+.book {
+  @include card-shadow;
+  display: flex;
+  flex-direction: column;
+  max-width: 20%;
+  padding: 32px 32px 20px 32px;
+
+  &-list {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    margin: 40px auto;
```

Comment:
If you use flexbox don´t need to position with margin auto, instead of use align-item

Reply:
Done 😄